campaignion_foundation.js

```diff
@@ -59,16 +65,20 @@ Drupal.behaviors.campaignion_foundation_clientside_validation.attach = function
 
   // Add classes to form elements.
   $('form', context).on('clientsideValidationInvalid', function (event, element) {
-    var $wrapper = findWrapper($(element));
-    $(element).addClass('is-invalid-input');
-    $wrapper.siblings('label').addClass('is-invalid-label');
-    $wrapper.next('.form-error').addClass('is-visible');
+    if (element) {
```

Comment:
Just out of curiosity: When is the `element` empty here?

Reply:
Clientside validation triggers the event 2x:
1. on the form → no `element`
2. on the element itself with `element` being the element